src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -371,15 +371,15 @@ var DynamicTable = new Class({
     },
 
     loadColumnsOrder: function() {
-        var columnsOrder = [];
-        var val = localStorage.getItem('columns_order_' + this.dynamicTableDivId);
+        const columnsOrder = [];
+        const val = localStorage.getItem('columns_order_' + this.dynamicTableDivId);
         if (val === null || val === undefined) return;
         val.split(',').forEach(function(v) {
             if ((v in this.columns) && (!columnsOrder.contains(v)))
                 columnsOrder.push(v);
         }.bind(this));
 
-        for (var i = 0; i < this.columns.length; ++i)
+        for (let i = 0; i < this.columns.length; ++i)
```

Comment:
```ReferenceError: assignment to undeclared variable i - dynamicTable.js:386:14```

Because this variable is now scoped properly, you must redeclare `i` in the for loop just below.

Reply:
@Piccirello, can you confirm that it's fixed now?

Reply:
I'm having some issues with my dev box, I'll need to test tomorrow.

Reply:
Ok, was able to test again. All is working well for me.